service/manager.go

```diff
@@ -385,25 +385,25 @@ func (m *manager) startModules() []error {
 	// calls to return
 	wg.Add(len(m.moduleWrappers))
 	for _, mod := range m.moduleWrappers {
-		go func(m *moduleWrapper) {
-			if !m.IsRunning() {
+		go func(modWrapper *moduleWrapper) {
```

Comment:
why do you need this change?
https://github.com/golang/go/wiki/CodeReviewComments#variable-names